### ISSUE TITLE
Name of deleted item is not read from people picker

### ISSUE THREAD

**Issue by rebeccaballantyne** [OfficeDev/office-ui-fabric-react#4817]
Are you willing to submit a PR to fix? No

Requested priority: Normal

Products/sites affected: ODB Share

#### Describe the issue:

When deleting selected items from the people picker (either by using backspace key or the "remove" button), the name of the person that was just removed should be read.

**Comment by cliffkoh** [comment]
Untag 6.0 tag as the 6.0.0 release isn't tracking any changes to People Picker specifically at the moment.

**Comment by JasonGore** [comment]
This was highlighted in #4817 as still being an outstanding issue.

**Comment by dzearing** [comment]
@joschect This is an accessibility issue opened May 2018. Isn't this a concern for the picker scenarios? Or was it fixed?

Jason mentioned above that #4817 is saying it's still an issue. Accessibility bugs need to be prioritized.

**Comment by JasonGore** [comment]
Hmm, my comment is creating a circular reference to this same issue. Let me try and see if I can remember what I meant.

**Comment by JasonGore** [comment]
I can only imagine I meant to link #5098, but that was closed as a dupe against this one. I would consider this issue still unresolved.

**Comment by joschect** [comment]
This should have been fixed, it doesn't read "deleted" or anything like that, but it will read the new selected list of names.

**Comment by dzearing** [comment]
Alright, closing.
